src/slices/TextCarousel/TextCarousel.tsx

```diff
@@ -48,29 +62,83 @@ export const TextCarousel: React.FC<TextCarouselProps> = ({
       </Wrapper>
 
       <CarouselContainer ref={containerRef}>
-        <CarouselInnerContainer
-          drag="x"
-          dragConstraints={containerRef}
-          numberofitems={slice.slides.length}
-        >
-          {slice.slides.map(({ id, title, text, icon }) => (
-            <CardContainer key={id} numberofitems={slice.slides.length}>
-              <TextCardWithIcon
-                title={title}
-                text={text}
-                icon={
-                  <Image
-                    src={strapiMediaUrl(icon.img, 'small')}
-                    alt={icon.alt}
-                    fill
-                    style={{ objectFit: icon.objectFit || 'contain' }}
+        <Box position="relative" width="full">
+          <CarouselInnerContainer
+            numberofitems={slice.slides.length}
+            animate={{
+              x: slideWidth * -sliderIndex,
+            }}
+            transition={{
+              duration: 0.3,
+              ease: 'easeInOut',
+            }}
+          >
+            {slice.slides.map(({ id, title, text, icon }) => (
+              <CardContainer
+                key={id}
+                ref={slideRef}
+                numberofitems={slice.slides.length}
+              >
+                <TextCardWithIcon
+                  title={title}
+                  text={text}
+                  icon={
+                    <Image
+                      src={strapiMediaUrl(icon.img, 'small')}
+                      alt={icon.alt}
+                      fill
+                      style={{ objectFit: icon.objectFit || 'contain' }}
+                    />
+                  }
+                  displayAs="column"
+                />
+              </CardContainer>
+            ))}
+          </CarouselInnerContainer>
+
+          <ButtonsContainer show={allowScroll.toString()}>
+            <Box>
+              <AnimatePresence>
+                {canMoveLeft && (
+                  <IconButton
+                    key="leftButton"
+                    as={motion.button}
+                    initial={{ opacity: 0 }}
+                    animate={{ opacity: 1 }}
+                    exit={{ opacity: 0 }}
+                    size="lg"
+                    variant="outline"
+                    onClick={() => setSliderIndex(sliderIndex - 1)}
+                    aria-label="Move left"
+                    icon={<ArrowLeft size={16} />}
+                    pointerEvents="auto"
+                    boxShadow="0px 4px 8px -1px rgba(0, 0, 0, 0.08), 0px 2px 4px -1px rgba(0, 0, 0, 0.06)"
```

Comment:
Would it be possible to have a similiar effect by using a token like `sm`, `md`, etc?
```suggestion
                    boxShadow="sm"
```